examples/redirectRuleExample.js

```diff
@@ -58,11 +58,11 @@ async function redirectRuleExample(user, context, callback) {
   }
 
   // Some kind of context check occurred to determine if a redirect should happen.
-  if (ruleUtils.canRedirect && !userUtils.setAppMeta("is_verified")) {
+  if (ruleUtils.canRedirect && !userUtils.getAppMeta("is_verified")) {
```

Comment:
I'm assuming this was a typo when the code was written. The intention here was to GET the current `app_metadata` value, right?